### ISSUE TITLE
Add Blue Slime Seed

### ISSUE THREAD

**Issue by dakamojo** [Darkosto/SkyFactory4Issues#546]
## Context

Suggest add a seed or other method to spawn a blue slime.  The player needs to manually kill 6 blue slimes to level up the data module.  A blue slime pristine matter is required for one of the achievements.

**Comment by Jauchi** [comment]
How do we get blue slimes at all? Are they disabled in the config file? 

We are currently in an ocean biome, trying to spawn these things in order to improve the data module.

We tried putting a bucket of blue slime over congluated blue slime, grass, slime grass, nothing worked.

We even built a seperate island in a swamp biome, covering the entire island so that other hostile mobs can spawn, but still no slimes.
We also tried the hunting dimension, no dice.

Any suggestions how to get blue slimes?
